pkg/debug/Dockerfile

```diff
@@ -3,11 +3,17 @@
 # has a fast path for stack unwinding. This also happens
 # to be a perfect place to put any other kind of debug info
 # into the package: see abuild/etc/abuild.conf.
-FROM alpine:3.12 as musl-build
+FROM lfedge/eve-alpine:99db0c3af59a9656315c9d7a0ad8a24f631134b0 as build
+ENV BUILD_PKGS abuild curl tar make linux-headers patch g++
+# Feel free to add additional packages here, but be aware that
+# EVE's rootfs image can be no larger than 300Mb
```

Comment:
Add "and you need to check that for all of the ISAs since arm64 binaries are typically larger and amd64 ones."?

Reply:
done